plasma_framework/contracts/src/vaults/ZeroHashesProvider.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.0;
+
+library ZeroHashesProvider {
+    // This is some optimization while building merkle tree for deposit block
+    // Pre-compute the essential data once and reuse in every deposit
```

Comment:
This comment probably belongs in the caller of `getZeroHashes()`, rather than here in the library.

Reply:
I've put more informative `@dev` comment here. I don't feel we need a comment in the caller, but if it really looks unclear to you, then I'll put a comment there. Just let me know.